spec/requests/api/v1/users_spec.rb

```diff
@@ -1 +1,61 @@
+require 'rails_helper'
 
+RSpec.describe 'Api::V1::Users', type: :request do
+  let(:json_response) { JSON.parse response.body, symbolize_names: true }
+  let(:serializer) { Api::V1::UserBlueprint }
+
+  describe 'get an user' do
+    let!(:user) { create :user }
+
+    subject { get api_v1_user_path(user_id) }
```

Comment:
I prefer todo it inside a method to make it clearer, because the subject in this context is the controller, so the call to the controller shouldn't be with this name.
```
def get_call(param_id)
  get :show, params: param_id
end